magik-language-server/src/main/java/nl/ramsolutions/sw/magik/languageserver/MagikLanguageServerSettings.java

```diff
@@ -78,4 +80,23 @@ public Path getChecksOverrideSettingsPath() {
     final String overrideConfigFile = this.properties.getPropertyString(OVERRIDE_CONFIG_FILE);
     return overrideConfigFile != null ? Path.of(overrideConfigFile) : null;
   }
+
+  /**
+   * Get magik.lint.runChecksOnSave.
+   *
+   * @return magik.lint.runChecksOnSave
+   */
+  public boolean runChecksOnSave() {
+    return this.properties.getPropertyBoolean(RUN_CHECKS_ON_SAVE, false) == Boolean.TRUE;
+  }
+
+  /**
+   * Get magik.lint.runChecksOnChangeDelay.
+   *
+   * @return magik.lint.runChecksOnChangeDelay
+   */
+  @CheckForNull
+  public Integer getRunChecksOnChangeDelay() {
+    return this.properties.getPropertyInteger(RUN_CHECKS_ON_CHANGE_DELAY);
```

Comment:
```suggestion
    return this.properties.getPropertyInteger(RUN_CHECKS_ON_CHANGE_DELAY, 300);
```

As stated in the package.json for the VS Code client.

Reply:
Actually, my intention was to disable this by default, hence the `@CheckForNull` annotation. But perhaps enabling it by default might be a good idea.